tools/migration/migrate_state.py

```diff
@@ -73,8 +69,12 @@ def get_state_manager_from_config(config_path, working_dir):
     config_manager = manager.ConfigManager(config_path)
     config_container = config_manager.load()
     state_config = config_container.get_master().state_persistence
-    with tool_utils.working_dir(working_dir):
+    cwd = os.getcwd()
```

Comment:
I kinda liked this context manager.
do you think the try/finally is more technically correct?

Reply:
yeah  i can keep it. i just didnt like having a whole separate file for it. i'll move directly to utils/__init__.py